src/Tasks/Common/Resources/Strings.resx

```diff
@@ -660,6 +660,10 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1138: The target framework '{0}' is out of support and will not receive security updates in the future. Please refer to {1} for more information about the support policy.</value>
     <comment>{StrBegin="NETSDK1138: "}</comment>
   </data>
+  <data name="WorkloadIsEol" xml:space="preserve">
+    <value>NETSDK1138: The workload '{0}' is out of support and will not receive security updates in the future. Please refer to {1} for more information about the support policy.</value>
```

Comment:
So far I think the intention was for each unique error message to have a unique code.  Do we want to try to re-use the existing code here?

@baronfel

Reply:
Do I need to pick an unused number? `NETSDK1202` I guess?